dist/arc.js

```diff
@@ -1,115 +1,115 @@
 'use strict';
 
 Object.defineProperty(exports, "__esModule", {
-  value: true
+    value: true
```

Comment:
i prefer 2 spaces myself

Reply:
LOL. We just moved from 2 to 4 in the solidity :)

Reply:
hmm, that is I set lint to -- 2.

Reply:
@tibetsprague You should look at the code in the PR in arc-js, not here.

Reply:
This PR is closed and the code changes obsolete, at least with respect to lint-related improvements.

Reply:
@tibetsprague  Do you recommend 4 then, or 2?  I don't have a strong preference.

Reply:
I much prefer 2 spaces. but im also ok with whatever